C/jets.c

```diff
@@ -79,3 +79,25 @@ bool sha_256_block(frameItem* dst, frameItem src, const txEnv* env) {
   write32s(dst, h, 8);
   return true;
 }
+
+/* parse_sequence : TWO^32 |- TWO^32 + TWO^32 */
+bool parse_lock(frameItem* dst, frameItem src, const txEnv* env) {
+  (void) env; // env is unused;
+  uint_fast32_t nLockTime = read32(&src);
+  writeBit(dst, 500000000U <= nLockTime);
+  write32(dst, nLockTime);
+  return true;
+}
+
+/* parse_sequence : TWO^32 |- TWO^16 + TWO^16 */
```

Comment:
In e9869ba76a6c820161b8b8696c89f417ef1f5ac0:

I believe this comment should be `TWO^32 |- S(TWO^16 + TWO^16)` (not sure if my notation is right here). It looks like the actual typing code is correct, it's only the comment that's off.

Reply:
Fixed.